packages/storybook/src/stories/grid-rows.stories.tsx

```diff
@@ -688,10 +698,31 @@ export function ValidateEditValueServerSide() {
         clearTimeout(promiseTimeout);
         clearTimeout(keyStrokeTimeoutRef.current);
 
-        apiRef.current.setEditCellProps({ id, field, props: { ...props, error: true } });
+        let oldModel = apiRef.current.getEditRowsModel();
```

Comment:
Using "old" feels strange. It's the current value of the model at the time it's called. Maybe we should reverse the tradeoff? Call "new" for the new value we aim to push?

Reply:
I updated everything to `newModel`.